wrapper/src/main/java/software/amazon/jdbc/plugin/failover/FailoverConnectionPlugin.java

```diff
@@ -137,15 +138,19 @@ public class FailoverConnectionPlugin extends AbstractConnectionPlugin {
           "enableClusterAwareFailover", "true",
           "Enable/disable cluster-aware failover logic");
 
+  public static final AwsWrapperProperty FAILOVER_STRICT_READER =
+      new AwsWrapperProperty(
+          "failoverStrictReader", "false",
+          "Enable/disable failover to a writer node for read-only connection.");
```

Comment:
I think your other description above is a bit clearer:
"Set to true to only allow failover to reader nodes during the reader failover process. If enabled, reader failover to a writer node will only be allowed for single-node clusters. This logic mimics the logic of the Aurora read-only cluster endpoint."